### DISCUSSION TITLE
GSoC 2024

### DISCUSSION THREAD

**Discussion by simeetnayan81** [pytorch/ignite#3182] [General]
Hi everyone. Google Summer of Code 2024 has been announced. Are there any initial draft ideas for pytorch-ignite this year? @vfdev-5 @sdesrozis

**Comment by vfdev-5** [comment]
@simeetnayan81 thanks for asking! Yes, pytorch-ignite will apply to GSoC2024 under NumFOCUS umbrella this year. Project ideas will be provided later. The ideas from the previous year can be seen here: https://github.com/pytorch/ignite/wiki
